### PR TITLE
Minor fixes from cherrypicks

### PR DESCRIPTION
**Pull request by aakatz3**
These are a few very very minor, common fixes from the cherrypicks branch. If these can be merged so I don't need to make them again, and so they arent forgotten, that would be ideal.

### CODE REVIEW

File: LICENSE_HEADER [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Copyright (c) 2020 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
```
Comment (ZachyFoxx) [inline LICENSE_HEADER]: license header is not a javadoc, do not treat it as such please.

File: src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) 2020 DumbDogDiner <a href="dumbdogdiner.com">&lt;dumbdogdiner.com&gt;</a>. All rights reserved.
```
Comment (ZachyFoxx) [inline src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: not a javadoc.

**Review by ZachyFoxx — CHANGES_REQUESTED** [review]

File: src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java (hunk 2 above)
Comment (aakatz3) [reply src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: It crashes the javadoc thing, if I remove those, then we need to change it so it inserts as a comment and not as javadoc.

Comment (ZachyFoxx) [reply src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: > It crashes the javadoc thing, if I remove those, then we need to change it so it inserts as a comment and not as javadoc.

cc: @Prouser123

Comment (aakatz3) [reply src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: https://github.com/hierynomus/license-gradle-plugin reviewing from here

Comment (aakatz3) [reply src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: there are a few other javadoc errors I'll fix in it too

Comment (jcxldn) [reply src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: > It crashes the javadoc thing, if I remove those, then we need to change it so it inserts as a comment and not as javadoc. 

@aakatz3 

When you removed the link from the file, did you also remove it from the LICENSE_HEADER file?￼

Comment (aakatz3) [reply src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: When it has the angle brackets (and not html escapes) javadoc gets mad. According to this and something it links, [https://github.com/hierynomus?tab=repositories], the fix should be:
```groovy
// License Plugin Options
license {
    header = project.file('LICENSE_HEADER')
    ext.year = Calendar.getInstance().get(Calendar.YEAR)
    mapping("java", "JAVAPKG_STYLE")
}
```
(I also made it put in the current year automatically)

The problem is that throws this error:
```
Execution failed for task ':licenseFormatMain'.
> The provided header type (javapkg_style) for java is invalid

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

```

Comment (aakatz3) [reply src/main/java/com/dumbdogdiner/stickyapi/annotation/Untested.java]: fixed it wait for push

**Review by ZachyFoxx — APPROVED** [review]

**Pull request merged**